README.md

```diff
@@ -9,7 +9,7 @@
 
 [![GitHub Issues](https://img.shields.io/github/issues/AlexSkrypnyk/shellvar.svg)](https://github.com/AlexSkrypnyk/shellvar/issues)
 [![GitHub Pull Requests](https://img.shields.io/github/issues-pr/AlexSkrypnyk/shellvar.svg)](https://github.com/AlexSkrypnyk/shellvar/pulls)
-[![Tests](https://github.com/AlexSkrypnyk/shellvar/actions/workflows/test.yml/badge.svg)](https://github.com/AlexSkrypnyk/shellvar/actions/workflows/test.yml)
+[![Tests](https://github.com/AlexSkrypnyk/shellvar/actions/workflows/test-php.yml/badge.svg)](https://github.com/AlexSkrypnyk/shellvar/actions/workflows/test-php.yml)
```

Comment:
Please fox references

Reply:
Hi @AlexSkrypnyk 
I removed test.yml and now using test-php as scaffold 1.120.0 
https://github.com/AlexSkrypnyk/shellvar/pull/44/files#diff-f8052f4d199aae28ac06bd45ba1354312518ab4342d74c77fb693b949fcd7dc3